main.go

```diff
@@ -6,3 +6,19 @@ func main() {
 
 	fmt.Println("Calculator v1.0")
 }
+
+func add(x int, y int) int {
+	return x + y
+}
+
+func sub(x int, y int) int {
```

Comment:
It would be nicer to use the full name in order to prevent confusion.
 
```suggestion
func subtract(x int, y int) int {
```